### PR TITLE
[server][test][DBTablesLastInfo] Fix invalid time stamp test value

### PR DESCRIPTION
**Pull request by cosmo0920**
Sorry, I've misunderstood about timestamp format.

### CODE REVIEW

File: server/test/testDBTablesLastInfo.cc [hunk 1]
```diff
@@ -68,7 +68,7 @@ void test_addLastInfo(void)
 	DECLARE_DBTABLES_LAST_INFO(dbLastInfo);
 	LastInfoDef lastInfo;
 	lastInfo.dataType = LAST_INFO_HOST;
-	lastInfo.value = "1432103640";
+	lastInfo.value = "201505151340.0000000000";
```
Comment (kz0817) [inline server/test/testDBTablesLastInfo.cc]: 0s are too many (10 zeros) !

**Comment by kz0817** [comment]
The previous format can also be accepted.

**Comment by cosmo0920** [comment]
Oh, I see.

BTW, `201505151340` is YYYYMMDDhhmm, lack of ss part, right?

**Comment by cosmo0920** [comment]
Ah..., I've misunderstood about TimeStamp specification. So, I close this PR.
I'm sorry to cause you trouble.

**Comment by kz0817** [comment]
> BTW, 201505151340 is YYYYMMDDhhmm, lack of ss part, right?

No, it's not right. I said wrong thing. The second part is necessary.

**Comment by cosmo0920** [comment]
I submitted a question for `TimeStamp` type.

**Pull request closed without merging**